src/renderer/components/pool/PoolCards.style.tsx

```diff
@@ -4,8 +4,11 @@ import styled from 'styled-components'
 const ITEM_GAP = '8px'
 
 export const Container = styled(A.Row)`
+  height: 100%;
   // Needed to compensate extra margin caused by last items of Col
-  margin-bottom: -${ITEM_GAP};
+  div:nth-last-child(-n + 2) {
+    margin-bottom: 0;
+  }
```

Comment:
could you pls set this value to the `Col` component itself (14th line) ? 
and also could you pls fix margins for mobile view ?
![image](https://user-images.githubusercontent.com/67793194/120184042-2d9a7400-c219-11eb-8fbd-9c7b98a65e43.png)